tools/compression/Makefile

```diff
@@ -0,0 +1,117 @@
+###########################################################################
```

Comment:
I think the compression algorithm must be under os/tools because all resources related to binary build process are present there. Please check and modify accordingly .

Reply:
ROMFS tool (TizenRT/tools/fs/mkromfsimg.sh) is called during build but is present outside os/tools.
I think we can keep the compression tool outside also, since there is precedent.

Reply:
@sunghan-chang 
Could you please tell us what is the criteria for placing something in the tools folder?
When should we use TizenRT/tools or os/tools ?